src/opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -115,6 +115,19 @@ message Span {
     // Indicates that the span covers the client-side wrapper around an RPC or
     // other remote request.
     CLIENT = 2;
+
+    // Indicates that the span is used internally. Default value.
+    INTERNAL = 3;
```

Comment:
there is no more `SPAN_KIND_UNSPECIFIED `. It was replaced with `INTERNAL`

Reply:
proto3 convention to have UNSPECIFIED for the 0 value. https://developers.google.com/protocol-buffers/docs/style#top_of_page look for enums.

Reply:
Semantic of unspecified is `INTERNAL`. So than get rid of explicit `INTERNAL`? Perhaps rename to `SPAN_KIND_INTERNAL_OR_UNSPECIFIED`?

Reply:
For now I left it as-is. I think this is more of a syntax restriction of proto. Happy to change it if we have a different consensus here.

Reply:
It will be easier if enum in language will match exactly the enum values in proto. Thus I was suggesting to make `INTERNAL` and `UNSPECIFIED` the same value. Also it eliminates question on how to process `UNSPECIFIED`.

For instance, if service recieves `UNSPECIFIED` - should it preserve the value or free to switch to `INTERNAL`. If need to preserve - Go SDK will need to expose the `UNSPECIFIED` as an option for creating `SpanData`

Reply:
SG, merged these two in https://github.com/open-telemetry/opentelemetry-proto/pull/9/commits/2e24a8bf3b023f3ea61eb63d7dc340633a85d8fd.

Reply:
I think UNSPECIFIED is proto3 specific because in proto3 they removed the notion of has value. I would suggest that we keep the UNSPECIFIED to be consistent with the proto3 recommendations.

About changing to INTERNAL, that is probably a valid behavior, alternative we can simply reject the request if UNSPECIFIED by saying that this field is required.

Reply:
I'm not a proto expert. Why the proposed change is bad? What's the reason to keep them separate if semantically we want internal to be exactly when `UNSPECIFIED` is used for? (In code it will be the same. If you forgot to tell us what you want it to be - it will be `INTERNAL`)

Reply:
If we ever want to distinguish between not set and default value this will be impossible with the current approach, and as the language suggested this value should be considered not set not a default value.

Reply:
if you feel strongly about it - I'm fine. Perhaps put a comment saying that implementation can assume `INTERNAL` when received `UNDEFINED`.

Reply:
Reverted 2e24a8b and added a note in https://github.com/open-telemetry/opentelemetry-proto/pull/9/commits/03c06742b895249dd52b4d9ba2955446543ec427.